pkg/api/indexer/core.go

```diff
@@ -25,15 +25,38 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// AddProjectNamespace adds an index for core.ProjectNamespace to the given indexer.
-func AddProjectNamespace(ctx context.Context, indexer client.FieldIndexer) error {
-	if err := indexer.IndexField(ctx, &gardencorev1beta1.Project{}, core.ProjectNamespace, func(obj client.Object) []string {
+var (
+	// ProjectNamespaceIndexerFunc extracts the .spec.namespace field of a Project.
+	ProjectNamespaceIndexerFunc = func(obj client.Object) []string {
```

Comment:
Why such "strange" var constructs? Can't we use "regular" functions here? Same for other files in this package.

Reply:
Good question :smile: Fixed.